paper-radio-button.html

```diff
@@ -65,27 +68,32 @@
       }
 
       #radioContainer {
-        display: inline-block;
+        @apply(--layout-inline);
+        @apply(--layout-center-center);
         position: relative;
-        width: 16px;
-        height: 16px;
+        width: var(--calculated-paper-radio-button-size);
+        height: var(--calculated-paper-radio-button-size);
         vertical-align: middle;
       }
 
       #ink {
         position: absolute;
-        top: -16px;
-        left: -16px;
-        width: 48px;
-        height: 48px;
+        top: 50%;
+        left: 50%;
+        width: calc(2.66 * var(--calculated-paper-radio-button-size));
+        height: calc(2.66 * var(--calculated-paper-radio-button-size));
         color: var(--paper-radio-button-unchecked-ink-color, --primary-text-color);
         opacity: 0.6;
         pointer-events: none;
+        -webkit-transform: translate(-50%, -50%);
+        transform: translate(-50%, -50%);
       }
 
       :host-context([dir="rtl"]) #ink {
```

Comment:
If you remove this block and set `right: auto;` on `#ink` it will always be positioned with `top: 50%; left: 50%;`. paper-ripple makes the assumption that it should set `0` for all of top, right, bottom and left so that it will fill the container. `paper-radio-button` overrides this and sets `top: 50%; left: 50%;` so that it can use the transform to center the ripple without worrying about the size of the container or the ripple itself (which it has changed). However, this leaves both bottom and left set to 0 (normally not a problem) but when they're both set the directionality of the container determines which takes precedence. Setting `right: auto;` basically says "defer to left", even in RTL.